src/Message/ApplePayAuthorizeRequest.php

```diff
@@ -0,0 +1,462 @@
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setPemCertPath($value)
+    {
+        /**
+         * @var static
+         */
+        return $this->setParameter('pemCertPath', $value);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getKeyCertPath()
+    {
+        return $this->getParameter('keyCertPath');
+    }
+
+    /**
+     * A key certificate associated with your merchant ID.
+     * Example:
+     * setKeyCertPath('./certs/path_to_cert.key.pem');
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setKeyCertPath($value)
+    {
+        /**
+         * @var static
+         */
+        return $this->setParameter('keyCertPath', $value);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getKeyCertPassword()
+    {
+        return $this->getParameter('keyCertPassword');
+    }
+
+    /**
+     * Password associated with your key certificate, if applicable.
+     *
+     * @param string $url
+     * @return static
+     */
+    public function setKeyCertPassword($value)
+    {
+        /**
+         * @var static
+         */
+        return $this->setParameter('keyCertPassword', $value);
+    }
+
+
+    /**
+     * @return string
+     */
+    public function getValidationUrl()
+    {
+        return $this->getParameter('validationUrl');
+    }
+
+    /**
+     * Set the validation URL for the request
+     *
+     * @param string $url
+     * @return static
+     */
+    public function setValidationUrl($url)
+    {
+        /**
+         * @var static
+         */
+        return $this->setParameter('validationUrl', $url);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getMerchantIdentifier()
+    {
+        return $this->getParameter('merchantIdentifier');
+    }
+
+    /**
+     * Set the merchant identifier for the request
+     *
+     * @param string $url
+     * @return static
+     */
+    public function setMerchantIdentifier($value)
+    {
+        /**
+         * @var static
+         */
+        return $this->setParameter('merchantIdentifier', $value);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getDisplayName()
+    {
+        return $this->getParameter('displayName');
+    }
+
+    /**
+     * Sets the display name for the request
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setDisplayName($value)
+    {
+        /**
+         * @var static
+         */
+        return $this->setParameter('displayName', $value);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getApplicationType()
+    {
+        return $this->getParameter('applicationType');
+    }
+
+    /**
+     * Sets the application type or the "initiative" for the request
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setApplicationType($value)
```

Comment:
since you deleted all the business chat stuff, having the application type parameter isn't necessary because it will always just be "web". so i think you could just delete all mention of this parameter and pass "web" directly